src/auxiliary/auxiliary.jl

```diff
@@ -212,3 +212,20 @@ macro timed(timer_output, label, expr)
     val
   end
 end
+
+
+"""
+    init_p4est()
+
+Initialize p4est by calling `p4est_init` and setting the log level to `SC_LP_ERROR`.
+This function will check if p4est is already initialized
+and if yes, do nothing, thus it is safe to call it multiple times.
+"""
+function init_p4est()
+  if p4est_package_id()[] >= 0
+    return nothing
+  end
+
+  # Initialize p4est with log level ERROR to prevent a lot of output in AMR simulations
+  p4est_init(C_NULL, SC_LP_ERROR)
+end
```

Comment:
I'm not sure. One would need to know the value of, e.g., `SC_LP_DEBUG` before starting Trixi. That means, one has to `using P4est` anyway, so why don't they just call `p4est_init(C_NULL, SC_LP_DEBUG)` instead of setting an environment variable?

Reply:
I was thinking about setting the environment variable to the string "SC_LP_DEBUG", but you're right. There's not really a need for this.